packages/world/src/WorldProxy.sol

```diff
@@ -44,6 +40,8 @@ contract WorldProxy is Proxy {
     StorageSlot.getAddressSlot(IMPLEMENTATION_SLOT).value = newImplementation;
 
     emit IERC1967.Upgraded(newImplementation);
```

Comment:
is it okay that this is emitted even on a new deploy?

Reply:
Yes that's part of the spec here https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/ERC1967/ERC1967Proxy.sol#L26-L28